src/content/tools/doc-comment-references.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Documentation comment references
+short-title: Comment references
+description: Learn about doc comment references and their syntax.
+---
+
+Doc comments can contain references to various code elements. A library member
```

Comment:
Describing doc comment references as referring to "elements" makes sense to me and I believe is what we've always said, but is that because of its implementation ties to the analyzer and our familiarity with it? I'm not necessarily saying we shouldn't use the term, but I just want to make sure it's the best term to use.

I can find very few cases in user-facing docs where we use the term "element" like this. Never do we say "code element". In fact, we are careful to use the term as it already has a few meanings (elements in a collection, collection literal elements, HTML elements, etc).

As a side note, [`comment_references`](https://dart.dev/lints/comment_references) lint docs mention "in-scope identifiers".

@bwilkerson for thoughts based on writing diagnostic messages.

\cc @leafpetersen For other considerations :)

Reply:
I'll stick to "identifiers" instead of "code elements" for now. Good call. I think "element" might be more correct in other cases. For example, an _identifier_ is not referred to, a _class_ is, via it's _identifier_. It is messy though.